src/styles/_catalog.scss

```
@@ -20,3 +20,35 @@
     margin-right: 48px;
   }
 }
+
+@media (min-width: 1024px) {
+  .catalog {
+    display: grid;
+    min-width: calc(200px * 4 + 48px * 3);
+    grid-auto-flow: row;
+  }
+}
```

```suggestion
@media (min-width: 1024px) {
    min-width: calc(200px * 4 + 48px * 3);
}
```
Just move media into the catalog selector and don't repeat yourself.
Also, no need to duplicate the same properties all the time. Set them just for the catalog and in media set only properties that are changing.
Btw, you can use this property to handle amount of cards instead:
`grid-template-columns: repeat(2/3/4, 200px);`